### PR TITLE
fix: add instantiation params to Amplifier contract docs

### PR DESCRIPTION
**Pull request by ffe9f8**
- Add instantiation params to Amplifier contract docs
- Re-word chain introduction intro

Preview: https://axelar-docs-git-marty-chain-integration-intro-axelar-network.vercel.app/dev/amplifier/chain-integration/introduction

### CODE REVIEW

File: src/pages/dev/amplifier/chain-integration/introduction.mdx [hunk 1]
```diff
@@ -7,57 +7,64 @@ The Axelar Virtual Machine (AVM) and Amplifier are currently under active develo
...
-    - Get started with our [Message Relayer Example](https://github.com/axelarnetwork/axelar-examples/tree/main/examples/amplifier)
-
-
-The [chain integration tutorial](/dev/amplifier/chain-integration/integrate-a-chain) will guide you through this process within the current Amplifier DevNet. Contract deployment on Testnet or Mainnet will require governance proposal.
+
+To connect your chain to the Axelar network via the Interchain Amplifier, you’ll need to build or instantiate four smart contracts and one relayer service. The [chain integration tutorial](/dev/amplifier/chain-integration/integrate-a-chain) will guide you through this process on the current Amplifier devnet, which requires submitting a [whitelisting form](https://docs.google.com/forms/d/e/1FAIpQLSchD7P1WfdSCQfaZAoqX7DyqJOqYKxXle47yrueTbOgkKQDiQ/viewform) to the Interop Labs team at the end. Contract deployment on testnet or mainnet will require a governance proposal.
+
+### Source chain gateway contract
+
+- This is the gateway contract for your source chain, built using the same tech stack as that chain. It serves as the primary API for developers sending messages from your chain.
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration/introduction.mdx:15]: "same tech stack as that chain" is not quite right. You use the tech stack of the chain. AKA Ethereum isn't built in Solidity, but the tech stack for smart contracts on Ethereum is Solidity.

File: src/pages/dev/amplifier/chain-integration/introduction.mdx [hunk 2]
```diff
@@ -7,57 +7,64 @@ The Axelar Virtual Machine (AVM) and Amplifier are currently under active develo
...
-- **Message Relayer**
-    - A service built with any tech stack that listens for events on your chain and relays them to the Axelar network.
-    - Get started with our [Message Relayer Example](https://github.com/axelarnetwork/axelar-examples/tree/main/examples/amplifier)
-
-
-The [chain integration tutorial](/dev/amplifier/chain-integration/integrate-a-chain) will guide you through this process within the current Amplifier DevNet. Contract deployment on Testnet or Mainnet will require governance proposal.
+
+To connect your chain to the Axelar network via the Interchain Amplifier, you’ll need to build or instantiate four smart contracts and one relayer service. The [chain integration tutorial](/dev/amplifier/chain-integration/integrate-a-chain) will guide you through this process on the current Amplifier devnet, which requires submitting a [whitelisting form](https://docs.google.com/forms/d/e/1FAIpQLSchD7P1WfdSCQfaZAoqX7DyqJOqYKxXle47yrueTbOgkKQDiQ/viewform) to the Interop Labs team at the end. Contract deployment on testnet or mainnet will require a governance proposal.
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration/introduction.mdx:11]: Any time we say there's an authorization step, we should explain that this is devnet-only, and that the connections in testnet and mainnet will be done via governance of the network.

File: src/pages/dev/amplifier/chain-integration/introduction.mdx [hunk 3]
```diff
@@ -7,57 +7,64 @@ The Axelar Virtual Machine (AVM) and Amplifier are currently under active develo
...
+
+### Source chain gateway contract
+
+- This is the gateway contract for your source chain, built using the same tech stack as that chain. It serves as the primary API for developers sending messages from your chain.
+- The contract will have to implement the [Cross-Chain Gateway Protocol](https://github.com/axelarnetwork/cgp-spec).
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration/introduction.mdx:16]: @milapsheth said that this is outdated and folks should instead implement the API of our EVM gateway.

**Review by StephenFluin — CHANGES_REQUESTED** [review]

**Comment by StephenFluin** [comment]
Is the title of this PR accurate? I'm confused because it's not about contract instantiation?

**Comment by ffe9f8** [comment]
Duplicated in current [chain integration intro](https://docs.axelar.dev/dev/amplifier/chain-integration/introduction).

**Pull request closed without merging**